pkg/custominjections/custominjections.go

```diff
@@ -0,0 +1,112 @@
+package custominjections
```

Comment:
I noticed there are some remainings when I updated the `custom injection` term to `user defined injection`. Would you mind updating this PR to use `user defined injection`? Ideally we would like to replace custom to user-defined.

Reply:
Done.